lib/html.js

```diff
@@ -625,6 +625,10 @@ exports.onerror = null;
  * @return {Node} DOM Node
  */
 var patch = exports.patch = function(elem, jml, filter) {
+  // Cannot patch text nodes
+  if (elem.nodeType === 3) {
```

Comment:
I know this is all over the codebase, but it might be nice to use the constant here (`Node.TEXT_NODE`) so you don't need the comment. I understand if you want to leave it so it's consistent though

Reply:
100% agree. I think constants would be a good refactor. went for consistency